Tools/ardupilotwaf/boards.py

```diff
@@ -227,6 +227,10 @@ def configure_env(self, cfg, env):
                       args = ['--libs', '--cflags'])
         env.LIB += cfg.env.LIB_LIBSYSTEMD
 
+        cfg.check_cfg(package='lttng-ust', mandatory=False, global_define=True,
+                      args = ['--libs', '--cflags'])
+        env.LIB += cfg.env['LIB_LTTNG-UST']
```

Comment:
I though it had to be something like that, but I wasn't aware of that issue in Python. Thanks.